src/taipy/core/cycle/cycle.py

```diff
@@ -132,6 +132,7 @@ def name(self, val):
     @property
     def properties(self):
         self._properties = _reload(self._MANAGER_NAME, self)._properties
+        self._properties._entity_owner = self
```

Comment:
I don't like this. It looks like a code smell to me.

Reply:
I won't argue with that 😅 but other than that, I'm not sure how to solve object references problem

Reply:
From what I saw, all the
```python
@property
    def properties(self):
```
are the same for all entities.
I think it makes sense to move it into the _Entity class.

And about the `_entity_owner` reference, maybe it is cleaner if we can do something inside the `_reload()` method. But I'm not sure if it is possible or not.